.github/actions/install-oppia-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup python by installing wheel
+      run: pip install wheel==0.35.0
+      shell: bash
+    - name: Check Yarn Cache
+      run: |
+        pwd
+        cd ..
+        pwd
+        echo 'cache-folder "yarn_cache"' > .yarnrc
+        yarn cache list
+        rm .yarnrc
+        cd -
+      shell: bash
+    - name: Install third party
+      run: python -m scripts.install_third_party_libs
+      shell: bash
+    - name: Check Yarn Cache
+      run: yarn cache list
+      shell: bash
```

Comment:
Any specific reason for this logging?

Reply:
We print the cache list before and after installing yarn dependencies. This lets us see what packages we got from the cache and what packages were freshly downloaded. I think this information will help debug any future problems with this caching

Reply:
COnsidering intall_third_party libs will log all the newly downloaded libs, do you think we still need to log here (And on top)?

Reply:
Where does `install_third_party_libs` log the newly downloaded yarn modules in this example log (does not include this PR's changes): https://github.com/oppia/oppia/runs/2758276107?check_suite_focus=true? I see python packages being installed and dependencies being installed to `oppia_tools/` and `third_party`, but no JS dependencies, which would be installed to `node_modules`.